docs/source/torch.rst

```diff
@@ -152,7 +152,7 @@ us to use the current accelerator as the default device for relevant concepts su
 Stream device_type, FSDP, etc.
 
 As of today, accelerator devices are (in no particular order) :doc:`"CUDA" <cuda>`, :doc:`"MTIA" <mtia>`,
-:doc:`"XPU" <xpu>`, :doc:`"MPS" <mps>`, "HPU", and PrivateUse1 (many device not in the PyTorch repo itself).
+:doc:`"XPU" <xpu>`, :doc:`"MPS" <mps>`, "HPU", "HIP", and PrivateUse1 (many device not in the PyTorch repo itself).
```

Comment:
note that HIP is not really used by anyone afaik as amd integration re-uses the cuda name, so not really important to have here.

Reply:
Got it, remove it to avoid ambiguity